gateway/auth_manager.go

```diff
@@ -63,15 +63,15 @@ func (b *DefaultSessionManager) ResetQuota(keyName string, session *user.Session
 	}).Info("Reset quota for key.")
 
 	rateLimiterSentinelKey := RateLimitKeyPrefix + keyName + ".BLOCKED"
+
 	// Clear the rate limiter
-	go b.store.DeleteRawKey(rateLimiterSentinelKey)
+	b.store.DeleteRawKey(rateLimiterSentinelKey)
```

Comment:
I assume the boolean returned is for if ops was successfully. could we use the return value(like logging). same for line 70, 74.